### ISSUE TITLE
GitLab CI broken: Lintian failure because of binary in /etc (profstats)

### ISSUE THREAD

**Issue by kmk3** [netblue30/firejail#4733]
Likely caused by commit 8f93df99f ("install profstats in /etc/firejail
directory -  undocumented, used only for development", 2021-11-30).

Failed pipelines for that commit:

* `build_debian_package`

<https://gitlab.com/Firejail/firejail_ci/-/jobs/1833634608>

```plaintext
*****************************************
install size 6368	firejail-0.9.67/debian
*****************************************
dpkg-deb: building package 'firejail' in 'debian.deb'.
warning: the authors of lintian do not recommend running it with root privileges!
E: firejail: binary-in-etc etc/firejail/profstats
W: firejail: executable-is-not-world-readable usr/lib/firejail/fcopy 0711
W: firejail: executable-is-not-world-readable usr/lib/firejail/fldd 0711
W: firejail: executable-is-not-world-readable usr/lib/firejail/fnet 0711
W: firejail: executable-is-not-world-readable usr/lib/firejail/fnetfilter 0711
W: firejail: executable-is-not-world-readable usr/lib/firejail/fsec-optimize 0711
W: firejail: executable-is-not-world-readable usr/lib/firejail/fsec-print 0711
W: firejail: executable-is-not-world-readable usr/lib/firejail/fseccomp 0711
W: firejail: executable-is-not-world-readable usr/lib/firejail/fshaper.sh 0711
N: 1 tag overridden (1 warning)
Makefile:224: recipe for target 'deb' failed
make: *** [deb] Error 1
Cleaning up project directory and file based variables 00:01
ERROR: Job failed: exit code 2
```

* `debian_ci`

<https://gitlab.com/Firejail/firejail_ci/-/jobs/1833634618>

```plaintext
make[1]: Entering directory '/builds/Firejail/firejail_ci'
dh_missing -pfirejail --fail-missing
dh_missing: warning: etc/firejail/profstats exists in debian/tmp but is not installed to anywhere 
dh_missing: error: missing files, aborting
	The following debhelper tools have reported what they installed (with files per package)
	 * dh_install: firejail (44), firejail-profiles (1174)
	 * dh_installdocs: firejail (5), firejail-profiles (0)
	 * dh_installman: firejail (0), firejail-profiles (0)
	If the missing files are installed by another tool, please file a bug against it.
	When filing the report, if the tool is not part of debhelper itself, please reference the
	"Logging helpers and dh_missing" section from the "PROGRAMMING" guide for debhelper (10.6.3+).
	  (in the debhelper package: /usr/share/doc/debhelper/PROGRAMMING.gz)
	Be sure to test with dpkg-buildpackage -A/-B as the results may vary when only a subset is built
	If the omission is intentional or no other helper can take care of this consider adding the
	paths to debian/not-installed.
make[1]: *** [debian/rules:34: override_dh_missing] Error 25
make[1]: Leaving directory '/builds/Firejail/firejail_ci'
make: *** [debian/rules:8: binary] Error 2
dpkg-buildpackage: error: debian/rules binary subprocess returned exit status 2
debuild: fatal error at line 1182:
dpkg-buildpackage -us -uc -ui -i -I failed
gbp:error: 'debuild -i -I -uc -us' failed: it exited with 29
Cleaning up project directory and file based variables 00:00
ERROR: Job failed: exit code 1
```

Cc: @netblue30 @reinerh

**Comment by rusty-snake** [comment]
rpmlint: `firejail.x86_64: E: binary-in-etc /etc/firejail/profstats`

Probably every distro will need to patch this out because it violates packaging-guidelines.

**Comment by reinerh** [comment]
Yes, this should really not get installed there. LIB(EXEC)DIR is probably a better place. Maybe it can also be "hidden" behind the configure flag that enables installation of contrib scripts?

**Comment by netblue30** [comment]
I'll put a fix in, thanks.

**Comment by kmk3** [comment]
Fixed by commit 30e754610 ("profstats fix (#4733)", 2021-12-10).

The GitLab pipeline runs successfully now:

* <https://gitlab.com/Firejail/firejail_ci/-/pipelines/427033924>
